lib/Main.coffee

```diff
@@ -79,6 +80,7 @@ module.exports =
         @providers.push(new KeywordProvider(@configuration))
         @providers.push(new DocblockAnnotationProvider(@configuration))
         @providers.push(new DocblockTagProvider(@configuration))
+        @providers.push(new PHPUnitTagProvider(@configuration))
```

Comment:
Could you add a package setting that toggles this provider? I agree that adding PHPUnit annotations can be very handy for those that need them, but some users might be using PHPSpec or other testing tools and may not want to see these results.